code/datums/position_point_vector.dm

```diff
@@ -130,6 +130,8 @@
 	return MODULUS(y, world.icon_size) - 16 - 1
 
 /datum/point/proc/mapcheck()
+	return FALSE	//Space's Enter() proc does this for us.
+	/*

```

Comment:
Do not leave commented-out code around, this is 2018, we have version control. If you're not sure if it works, test it.